### DISCUSSION TITLE
創建php運行環境失敗

### DISCUSSION THREAD

**Discussion by cowcfj** [1Panel-dev/1Panel#4602] [Q&A]
主機類型：oracle cloud arm64
os：ubuntu

出現錯誤：
鏡像 build 失敗:failed to solve: process "/bin/sh -c if [ $CONTAINER_PACKAGE_URL ] ; then sed -i \"s/dl-cdn.alpinelinux.org/${CONTAINER_PACKAGE_URL}/g\" /etc/apk/repositories ; fi" did not complete successfully: Address already in use

**Comment by wanghe-fit2cloud** [comment]
> 主機類型：oracle cloud arm64 os：ubuntu
> 
> 出現錯誤： 鏡像 build 失敗:failed to solve: process "/bin/sh -c if [ CONTAINERPACKAGEURL];thensed−i"s/dl−cdn.alpinelinux.org/{CONTAINER_PACKAGE_URL}/g" /etc/apk/repositories ; fi" did not complete successfully: Address already in use

安装运行环境时，尝试切换一下扩展源。

**Reply by cowcfj** [reply]
所有擴展源都試過了，都失敗。

**Reply by wanghe-fit2cloud** [reply]
容器内是否正常访问外网？
> 可以通过 docker run 命令来创建一个 Nginx 容器来测试一下。

**Reply by cowcfj** [reply]
安裝了軟件商店的openresty，一直都可通過它搭建反代來訪問容器

**Reply by wanghe-fit2cloud** [reply]
反代容器没有问题，目前要排查一下容器内是否能正常访问外网。
